### ISSUE TITLE
Increase text-to-background contrast of the trigger buttons

### ISSUE THREAD

**Issue by pomerantsev** [pomerantsev/accented#121]
The current `white` text on `red` background fails minimum contrast per WCAG.

<img width="272" alt="Image" src="https://github.com/user-attachments/assets/ed616487-c8b2-4c93-a334-fe1635a45589" />
